llvm/test/CodeGen/X86/StackColoring.ll

```diff
@@ -581,6 +581,41 @@ onerr:
 
 %Data = type { [32 x i64] }
 
+declare void @throw()
```

Comment:
Refactoring change shouldn't have test changes?

Reply:
This commit was removed.